src/components/auth/node.tsx

```diff
@@ -140,6 +130,40 @@ export function Node({
   })
   return null
 
+  function renderInput(attributes: UiNodeInputAttributes) {
+    const basicFields = {
+      className:
+        'text-xl serlo-input-font-reset serlo-button-light hover:bg-brand-150 focus:bg-brand-150 outline-none -ml-1 mt-1 text-brand hover:text-brand px-4 py-2 w-full',
+      name: attributes.name,
+      onChange: (e: { target: { value: string } }) => {
+        void onChange(e.target.value)
+      },
+    }
+
+    if (attributes.name === 'traits.interest') {
+      return (
+        <select {...basicFields}>
+          <option value="nothing">{strings.auth.interests.pleaseChoose}</option>
```

Comment:
@hugotiburtino can we use an empty string if nothing is selected?

Reply:
Hey @elbotho,
I've put a comment in 91bd831 (which was for some reason not pushed before, although my local git was saying it was, who know? Maybe gh cli bug...)
If you are uncomfortable with it, I could switch in [CF Worker](https://github.com/serlo/cloudflare-worker/pull/380/files) (add 'nothing'  there, in order to let frontend set it to ''). 

The only important things are: 
- some legacy users in the DB just don't have interest field (I supposed it was only later made required). We have to support it.
- In order to continue requiring the users to fill it in, value null/nothing/empty here and in CF should be different.

Reply:
Thanks for the explanation.
So I could replace `nothing` with `–` here? I'd just prefer something obviously different.

Reply:
> So I could replace nothing with – here? 

sure

Reply:
@hugotiburtino In 3ac4774a76e17839b5fa6b6c400744fb9add415a I tried to find a default way of solving this.
Since it set to required it will actually not send a value for `interest` at all which seems to work fine with the backend.

Styling explained [here](https://www.w3docs.com/snippets/css/how-to-create-a-placeholder-for-an-html5-select-box-by-using-only-html-and-css.html) (second approach)